src/services/api/media.js

```diff
@@ -18,31 +14,18 @@ const media = {
     const nextPage = response?.next
     return { ...response, nextPage, lastPage }
   },
+  getAudio: async ({ sitename, id }) =>
+    apiBase.get(`${SITES}/${sitename}/${AUDIO_PATH}/${id}`).json(),
+  getImage: async ({ sitename, id }) =>
+    apiBase.get(`${SITES}/${sitename}/${IMAGE_PATH}/${id}`).json(),
+  getVideo: async ({ sitename, id }) =>
+    apiBase.get(`${SITES}/${sitename}/${VIDEO_PATH}/${id}`).json(),
   getMediaDocument: async ({ sitename, docId, docType }) =>
     apiBase.get(`${SITES}/${sitename}/${docType}/${docId}`).json(),
   getS3Url: async () =>
```

Comment:
If we're no longer using the getS3Url service please feel free to remove it from here.